playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/post.yml

```diff
@@ -52,7 +52,7 @@
   - name: Update registry image to current version
     when: _default_registry.rc == 0
     command: >
-      {{ oc_cmd }} patch dc/docker-registry -p
+      {{ oc_cmd }} patch dc/docker-registry -p -n default
```

Comment:
@jkroepke it does not. I previously created this issue: https://github.com/openshift/openshift-ansible/issues/1904 to address the use of the default user kubeconfig for these operations.